internal/compiler/generator/rust.rs

```diff
@@ -2444,6 +2444,20 @@ fn compile_builtin_function_call(
         BuiltinFunction::Translate => {
             quote!(slint::private_unstable_api::translate(#(#a),*))
         }
+        BuiltinFunction::MapPointToWindow => {
+            if let [Expression::PropertyReference(pr), Expression::Struct { values, .. }] =
```

Comment:
I think that's error prone to assume that this will always be a Expression::Struct.
I can imagine some pass that optimize common sub expression and replace it with a local variable load.

I can see a few possibilities:
 1. keep it as is, but at least document that properly in `BuiltinFunction::MapPointToWindow`
 2. Use the real `LogicalPoint` type somehow  (like the one in builtins.slint) and just forward the compiled expression to `map_to_window`)
 3. use two argument x and y as Type::Float
 4. Remove the point alltogether and call it `BuiltinFunction::ItemAbsolutePosition` or `ItemWindowPosition`

Reply:
Agreed. I'll go with option number 4. That option had crossed my mind, too.